scripts/build/appveyor/configure_appveyor_VS2017.bat

```diff
@@ -11,7 +11,7 @@ set CC=cl.exe
 set CXX=cl.exe
 
 rem Set variables
-set KRATOS_BUILD_TYPE=Release
+set KRATOS_BUILD_TYPE=Debug
```

Comment:
is it worth updating appveyor?
I would not touch it any more

Reply:
In any case, lets keep things consistent, its just adding a couple of lines...